src/resolvers/guidance.ts

```diff
@@ -281,15 +213,18 @@ export const resolvers: Resolvers = {
     guidanceGroup: async (parent: Guidance, _, context: MyContext): Promise<GuidanceGroup> => {
       return await GuidanceGroup.findById('Chained Guidance.guidanceGroup', context, parent.guidanceGroupId);
     },
-    // Chained resolver to fetch the Tags for this Guidance
-    tags: async (parent: Guidance, _, context: MyContext): Promise<Tag[]> => {
-      return await Tag.findByGuidanceId('Chained Guidance.tags', context, parent.id);
-    },
     created: (parent: Guidance) => {
       return normaliseDateTime(parent.created);
     },
     modified: (parent: Guidance) => {
       return normaliseDateTime(parent.modified);
     },
-  }
+    // Resolver to get the user who last modified this guidance
+    user: async (parent: Guidance, _, context: MyContext): Promise<User | null> => {
```

Comment:
this is nit-picky, but can we change this to be `modifiedBy`? I think it makes the relationship to the user clearer